lib/lunchbot_web/live/select_menu_live.ex

```diff
@@ -1,18 +1,26 @@
-defmodule LunchbotWeb.Admin.SelectMenuController do
-  use LunchbotWeb, :controller
+defmodule LunchbotWeb.Admin.SelectMenuLive do
+  use LunchbotWeb, :live_view
 
+  # alias LunchbotWeb.Router.Helpers, as: Routes
```

Comment:
This line is commented out.  Should it be removed, or is it serving a purpose?

Reply:
It should be removed. I forgot to do that. Nice catch!